symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -3,18 +3,27 @@ bdk:
   host: acme.symphony.com
   bot:
     username: mybot
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/keys/rsa-private-bot.pem
```

Comment:
Let's try to follow a unique convention. RSA private key is always located under `${user.home}/.symphony/privatekey.pem`